src/interface/meta.thrift

```diff
@@ -687,6 +687,17 @@ struct ListIndexStatusResp {
     3: list<IndexStatus>    statuses,
 }
 
+struct CopySchemaFromSpaceReq {
+    1: string         current_space,
+    2: string         from_space,
+    3: bool           need_index,
```

Comment:
if the source space have create some index we should copy them into target space？

Reply:
The user can choose whether need. Default will do it